boofuzz/fuzz_logger_csv.py

```diff
@@ -36,7 +36,7 @@ class FuzzLoggerCsv(ifuzz_logger_backend.IFuzzLoggerBackend):
     def __init__(self, file_handle=sys.stdout, bytes_to_str=DEFAULT_HEX_TO_STR):
         """
         Args:
-            file_handle (io.TextIOBase): Open file handle for logging. Defaults to sys.stdout.
+            file_handle (io.TextIO): Open file handle for logging. Defaults to sys.stdout.
```

Comment:
`open('ftp-fuzz.csv', 'wb')` generates `_io.BufferedWriter` (BinaryIO in pycharm) which matches only TextIO and [BinaryIO](https://docs.python.org/3/library/io.html#binary-i-o).
As we don't need en-/decoding here (?), I suggest we use BinaryIO instead.

FileIO doesn't match either of them so creating a file handle as above would cause a missleading type warning in an IDE.

Reply:
I guess BinaryIO is appropriate, as the csv.writer docs say you should open files with the 'b' option: https://docs.python.org/2/library/csv.html#csv.writer